jetty-server/src/main/java/org/eclipse/jetty/server/Response.java

```diff
@@ -103,7 +104,7 @@
 
     private enum EncodingFrom
     {
-        NOT_SET, INFERRED, SET_LOCALE, SET_CONTENT_TYPE, SET_CHARACTER_ENCODING
+        NOT_SET, INFERRED, SET_LOCALE, SET_CONTENT_TYPE, SET_CHARACTER_ENCODING, DEFAULT
```

Comment:
Should not the new constant be called `CONTEXT`?

Reply:
I'm removing the constant, as we never set the encoding on the response explicitly if it is via the context default.